cmd/syncthing/main.go

```diff
@@ -625,7 +638,6 @@ func syncthingMain(runtimeOptions RuntimeOptions) {
 	// Ensure that we have a certificate and key.
 	cert, err := tls.LoadX509KeyPair(locations[locCertFile], locations[locKeyFile])
 	if err != nil {
-		l.Infof("Generating ECDSA key and certificate for %s...", tlsDefaultCommonName)
```

Comment:
Why?

Reply:
That was not intentional. I did add the line again.